src/components/Vault/BasicVault/VrtCard.tsx

```diff
@@ -0,0 +1,145 @@
+// We put the code of UI of old VAI pool (which will be live for quite some time) into this seperated
+// file, instead of merging its logic into general pool UI which is in `./Card.js` thus we can easily
+// remove this VAI pool code in the future when it's about to be deprecated
+import React, { useState, useEffect } from 'react';
+
+import BigNumber from 'bignumber.js';
+import { connect } from 'react-redux';
+import { useWeb3React } from '@web3-react/core';
+import { Setting } from 'types';
+import { State } from 'core/modules/initialState';
+import useRefresh from 'hooks/useRefresh';
+import { useToken, useVrtVaultProxy } from 'hooks/useContract';
+import { getVrtVaultProxyAddress } from 'utilities/addressHelpers';
+
+import CardContent from './CardContent';
+import CardHeader from './CardHeader';
+import { VaultCardWrapper } from '../styles';
+
+interface VaultCardProps {
+  settings: Setting;
+}
+
+const DAYS_OF_YEAR = 365;
+const BLOCK_PER_MINUTE = 60 / 3;
+const BLOCK_PER_DAY = BLOCK_PER_MINUTE * 60 * 24;
+
+function VaultCard({ settings }: VaultCardProps) {
```

Comment:
`settings` seems to be unused in the component, so I'd say remove it :)

Reply:
sure